### PR TITLE
Adding script command dependencies

### PR DESCRIPTION
**Pull request by sachintu47**

### CODE REVIEW

File: include/paths.h [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#ifndef _PATHS_H_
+
+#define        _PATH_BSHELL        "/bin/sh"
```
Comment (IgorTodorovskiIBM) [inline include/paths.h:3]: Is the extra spacing between define and _PATH_BSHELL intended? 

Can you also follow the header convention like this: https://github.com/ibmruntimes/zoslib/blob/main/include/locale.h#L1-L10

File: src/zos-pty.c [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    *slave = -1;
+
+    // Iterating through the 4-digit ptyp and ttyp devices
+    int i=0;
+    for (; i < 10000; i++) {  // Maximum 9999 for 4-digit format
+        snprintf(master_dev, sizeof(master_dev), "/dev/ptyp%04d", i);
+        snprintf(slave_dev, sizeof(slave_dev), "/dev/ttyp%04d", i);
+
+       if ((*master = open(master_dev, O_RDWR | O_NOCTTY)) != -1) {
+                if (grantpt(*master) != 0)
```
Comment (IgorTodorovskiIBM) [inline src/zos-pty.c:31]: Are you able to make the spacing match the indentation of the other files? (tabs are 2 spaces)

File: include/sys/time.h [hunk 3]
```diff
@@ -65,4 +65,32 @@ __Z_EXPORT int lutimes(const char *filename, const struct timeval tv[2]);
 
 #endif
 
+/* Convenience macros for operations on timevals.
+   NOTE: `timercmp' does not work for >= or <=.  */
+# define timerisset(tvp)        ((tvp)->tv_sec || (tvp)->tv_usec)
```
Comment (IgorTodorovskiIBM) [inline include/sys/time.h:70]:
```suggestion
#define timerisset(tvp)        ((tvp)->tv_sec || (tvp)->tv_usec)
```

File: include/sys/signalfd.h [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	int32_t ssi_status;
+	uint32_t ssi_int;
+	uint64_t ssi_ptr;
+	uint64_t ssi_utime;
+	uint64_t ssi_stime;
+	uint64_t ssi_addr;
+
+	/*
+	 * Pad strcture to 128 bytes. Remember to update the
+	 * pad size when you add new memebers. We use a fixed
```
Comment (IgorTodorovskiIBM) [inline include/sys/signalfd.h:45]:
```suggestion
	 * pad size when you add new members. We use a fixed
```

**Review by IgorTodorovskiIBM — COMMENTED** [review]
Very good additions! I have a couple of comments

File: src/zos-pty.c (hunk 2 above)
Comment (sachintu47) [reply src/zos-pty.c:31]: Indentation corrected

File: include/sys/time.h (hunk 3 above)
Comment (sachintu47) [reply include/sys/time.h]: Made suggested changes

File: include/sys/signalfd.h (hunk 4 above)
Comment (sachintu47) [reply include/sys/signalfd.h]: Corrected

File: include/sys/signalfd.h [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  {
+    SFD_CLOEXEC = 02000000,
+#define SFD_CLOEXEC SFD_CLOEXEC
+    SFD_NONBLOCK = 00004000
+#define SFD_NONBLOCK SFD_NONBLOCK
+  };
+
+
+struct signalfd_siginfo {
+	uint32_t ssi_signo;
```
Comment (IgorTodorovskiIBM) [inline include/sys/signalfd.h:26]: Use two spaces for indentation like https://github.com/ibmruntimes/zoslib/blob/main/src/zos-mount.c

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

File: include/sys/signalfd.h (hunk 5 above)
Comment (sachintu47) [reply include/sys/signalfd.h]: Done

**Review by HarithaIBM — APPROVED** [review]

**Pull request merged**